CMakeLists.txt

```diff
@@ -89,7 +89,7 @@ include( ROCMInstallSymlinks )
 include( ROCMClients )
 include( ROCMHeaderWrapper )
 
-set ( VERSION_STRING "1.9.0" )
+set ( VERSION_STRING "2.0.0" )
```

Comment:
why the major version update?

Reply:
I think any code written with just hipSOLVER shouldn't be affected by the change, but it's an API change that will affect any code written with both hipSOLVER and hipBLAS. We're breaking compatibility with previous versions of hipBLAS, so I thought that justified a major version update.

Reply:
Also, I believe hipBLAS is updating their major version number as well. This'll make it easier to tell users which versions of hipSOLVER and hipBLAS work together, i.e. hipSOLVER v2 goes together with hipBLAS v2, and doesn't work with hipBLAS v1.